env/container/go/Dockerfile

```diff
@@ -54,3 +54,6 @@ RUN apt-get update && apt-get install net-tools
 RUN go get -u github.com/githubnemo/CompileDaemon
 
 ENTRYPOINT CompileDaemon -log-prefix=false -build="go build -o /go/bin/trash_hunt_tcp_server ." -command="/go/bin/trash_hunt_tcp_server"
+
+RUN useradd -ms /bin/bash user
```

Comment:
This is exactly what I was thinking :+1: but I think we should do it up here

https://github.com/tylergeery/trash_hunt/pull/25/files#diff-8b98e42e272f15295feefb882af1a638R9

since that is as shared stage for the api, tcp, and migraiton docker environments

https://docs.docker.com/develop/develop-images/multistage-build/